packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx

```diff
@@ -0,0 +1,127 @@
+import classNames from 'classnames';
+import React from 'react';
+import { isCryptocurrency, getLimitOrderAmount, isValidToSell, addComma } from '@deriv/shared';
+import ContractCardItem from './contract-card-item';
+import ToggleCardDialog from './toggle-card-dialog';
+import Icon from '../../icon';
+import MobileWrapper from '../../mobile-wrapper';
+import Money from '../../money';
+import { ResultStatusIcon } from '../result-overlay/result-overlay';
+import { TGeneralContractCardBodyProps } from './contract-card-body';
+
+type TTurbosCardBody = Pick<
+    TGeneralContractCardBodyProps,
+    | 'addToast'
+    | 'connectWithContractUpdate'
+    | 'contract_info'
+    | 'contract_update'
+    | 'currency'
+    | 'current_focus'
+    | 'error_message_alignment'
+    | 'getCardLabels'
+    | 'getContractById'
+    | 'is_sold'
+    | 'onMouseLeave'
+    | 'removeToast'
+    | 'setCurrentFocus'
+    | 'status'
+> & {
+    progress_slider_mobile_el: React.ReactNode;
+};
+
+const TurbosCardBody = ({
+    contract_info,
+    contract_update,
+    currency,
+    getCardLabels,
+    is_sold,
+    progress_slider_mobile_el,
+    status,
+    ...toggle_card_dialog_props
+}: TTurbosCardBody) => {
+    const {
+        bid_price,
+        buy_price,
+        profit,
+        barrier,
+        entry_spot_display_value,
+        limit_order = {},
+        sell_price,
+    } = contract_info;
+    const { take_profit } = getLimitOrderAmount(contract_update || limit_order);
+    const is_valid_to_sell = isValidToSell(contract_info);
+    const contract_value = is_sold ? sell_price : bid_price;
+    const { BARRIER, CONTRACT_VALUE, ENTRY_SPOT, TAKE_PROFIT, TOTAL_PROFIT_LOSS, PURCHASE_PRICE } = getCardLabels();
+
+    return (
+        <React.Fragment>
+            <div className={classNames('dc-contract-card-items-wrapper dc-contract-card--turbos')}>
+                <ContractCardItem
+                    className='dc-contract-card__buy-price'
+                    is_crypto={isCryptocurrency(currency)}
+                    header={PURCHASE_PRICE}
+                >
+                    <Money amount={buy_price} currency={currency} />
+                </ContractCardItem>
+                <ContractCardItem header={CONTRACT_VALUE} className='dc-contract-card__contract-value'>
+                    <Money amount={contract_value} currency={currency} />
+                </ContractCardItem>
+                <ContractCardItem
+                    header={ENTRY_SPOT}
+                    is_crypto={isCryptocurrency(currency)}
+                    className='dc-contract-card__entry-spot'
+                >
+                    <Money amount={`${entry_spot_display_value}`} />
```

Comment:
```suggestion
                    <Money amount={entry_spot_display_value} />
```

Reply:
Thank you, didn't notice🙇🏼‍♀️

Reply:
Hi Jim! This was done because `amount` accepts `string | number | undefined` and `entry_spot_display_value` can be `string | null | undefined`